src/patterns/components/highlight-board/no-image.hbs

```diff
@@ -0,0 +1,6 @@
+---
+order: 2
+description: This variation of the highlight board component should be used when there is an alternative to a
+  photograph behind the lead message and cta. Examples of this are a wash of color or a pattern set behind the content.
```

Comment:
I think we should spell out "cta" as call to action once per component page and then we can use "CTA" after we define it at least once.